common/src/main/java/com/viaversion/viaversion/protocol/ProtocolManagerImpl.java

```diff
@@ -134,12 +134,12 @@ public ProtocolManagerImpl() {
 
     public void registerProtocols() {
         // Base Protocol
-        registerBaseProtocol(BASE_PROTOCOL, Range.lessThan(Integer.MIN_VALUE));
-        registerBaseProtocol(new BaseProtocol1_7(), Range.lessThan(ProtocolVersion.v1_16.getVersion()));
-        registerBaseProtocol(new BaseProtocol1_16(), Range.atLeast(ProtocolVersion.v1_16.getVersion()));
+        BASE_PROTOCOL.initialize();
```

Comment:
Forgot that this also needs:
```suggestion
        BASE_PROTOCOL.initialize();
        BASE_PROTOCOL.register(Via.getManager().getProviders());
```